kr_mesh_visualization/CMakeLists.txt

```diff
@@ -1,34 +1,41 @@
 cmake_minimum_required(VERSION 3.10)
 project(kr_mesh_visualization)
 
-# set default build type
-if(NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE RelWithDebInfo)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-add_compile_options(-Wall)
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(nav_msgs REQUIRED)
+find_package(visualization_msgs REQUIRED)
 
-find_package(catkin REQUIRED COMPONENTS geometry_msgs nav_msgs topic_tools visualization_msgs)
+add_executable(${PROJECT_NAME} src/mesh_visualization.cpp)
+ament_target_dependencies(${PROJECT_NAME} rclcpp geometry_msgs nav_msgs visualization_msgs)
 
-catkin_package(
-  CATKIN_DEPENDS
-  geometry_msgs
-  nav_msgs
-  topic_tools
-  visualization_msgs)
+install(DIRECTORY
+  mesh
```

Comment:
Why is this just mesh?

Reply:
This is to install the mesh directory as a whole to the package